libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestRequest.m

```diff
@@ -281,4 +275,16 @@ + (SFRestMethod)sfRestMethodFromHTTPMethod:(NSString *)httpMethod {
     return restMethodName;
 }
 
++ (NSString *)toQueryString:(NSDictionary *)components {
+    NSMutableString *params = [NSMutableString new];
+    if (components) {
+        [params appendString:@"?"];
+        for (NSString *paramName in [components allKeys]) {
+            [params appendString:paramName];
+            [params appendString:@"="];
+            [params appendString:[components[paramName] stringByURLEncoding]];
```

Comment:
Don't we need to append a `&` at the end of this block before we add more params?